server/build/webpack.js

```diff
@@ -49,7 +50,10 @@ export default async function createCompiler (dir, { hotReload = false } = {}) {
     new DynamicEntryPlugin(),
     new UnlinkFilePlugin(),
     new WatchRemoveEventPlugin(),
-    new WatchPagesPlugin(dir)
+    new WatchPagesPlugin(dir),
+    new FriendlyErrorsWebpackPlugin({
+      messages: ['Your application is running here http://localhost:3000']
```

Comment:
My bad, this shouldn't be here in the first place, as this message is not even displayed.
We already have a similar message displayed by bin/next-dev.js. Will append a commit to this PR.